config.json

```diff
@@ -69,7 +69,8 @@
     },
     "travis": {
       "blacklist": [
-        "repos/polymer-starter-kit"
+        "repos/polymer-starter-kit",
+        "repos/paper-checkbox"
```

Comment:
@abdonrd You could just commit the eslintrc.json file to your repo, but otherwise seems reasonable.